confluent_platform/README.md

```diff
@@ -10,7 +10,7 @@ This integration collects JMX metrics for the following components:
 - Connect
 - Replicator
 - Schema Registry
-- SQL Server
+- KSQLDB Server
```

Comment:
```suggestion
- ksqlDB Server
```
To match official naming on [Confluent](https://www.confluent.io/product/ksql)

Reply:
agreed. otherwise 👍  from docs